plugins/hls-hlint-plugin/hls-hlint-plugin.cabal

```diff
@@ -2,11 +2,11 @@ cabal-version: 2.2
 name:          hls-hlint-plugin
 version:       0.1.0.0
 synopsis:      Hlint integration plugin with Haskell Language Server
-description:   Please see README.md
+description:   Please see Haskell Language Server Readme (https://github.com/haskell/haskell-language-server/blob/master/README.md)
```

Comment:
https://github.com/haskell/haskell-language-server#readme is what i usually use

Reply:
oh, yeah, thanks